src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -13,11 +13,17 @@ namespace System.Device.Gpio.Drivers
 {
     public unsafe partial class RaspberryPi3Driver : GpioDriver
     {
+
+        // error indicates that an entity doesn't exist
+        private const int Linux_Enoent = 2;
+
         private RegisterView* _registerViewPointer = null;
-        private const int GpioRegisterOffset = 0x00;
         private static readonly object s_initializationLock = new object();
         private static readonly object s_sysFsInitializationLock = new object();
+        private const uint GpioPeripheralOffset = 0x0020_0000; // offset from the peripheral base address of the GPIO registers
```

Comment:
@buyaa-n does this look correct to you? asking because you mentioned once something about address being different on x64 vs x86 or something along these lines

Reply:
Have no idea about this address, that was about pin numbering offset address only for SysFsDriver, didn't tried RaspberryPiDriver for 64 bit Linux. Read the offset from the system https://github.com/dotnet/iot/pull/386/files#diff-f9b9a68ada4ceabc2a9b052f1c52a2cbR43
  
I guess he get this from device reference or something like that, so maybe better add link to reference

Reply:
This looks correct to me. This is in the BCM chipset datasheet and it is standard for all of those chips so it is accurate.